mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -59,6 +59,7 @@ private[classification] trait LogisticRegressionParams extends ProbabilisticClas
    *       equivalent.
    *
    * Default is 0.5.
+ *
```

Comment:
IDEA Scala plugin messes up with ScalaDoc when doing refactoring and import organization. Will revert these unexpected changes.